src/kernels/raw/session/rawSession.node.ts

```diff
@@ -62,7 +65,10 @@ export class RawSession implements ISessionWithSocket {
         this._kernelChanged = new signaling.Signal<this, Session.ISessionConnection.IKernelChangedArgs>(this);
         this._ioPubMessage = new signaling.Signal<this, KernelMessage.IIOPubMessage>(this);
         this._terminated = new signaling.Signal<this, void>(this);
+        this._anyMessage = new signaling.Signal<this, Kernel.IAnyMessageArgs>(this);
+        this._unhandledMessage = new signaling.Signal<this, KernelMessage.IMessage>(this);
```

Comment:
We were doing things correctly for some events, but not all, modified to ensure we do the same for all events, as this is required for things to work correctly.